src/main/java/uk/gov/hmcts/darts/audio/repository/MediaRequestRepository.java

```diff
@@ -4,10 +4,7 @@
 import org.springframework.stereotype.Repository;
 import uk.gov.hmcts.darts.audio.entity.MediaRequest;
 
-import java.util.List;
-
 @Repository
-public interface AudioRequestRepository extends JpaRepository<MediaRequest, Integer> {
+public interface MediaRequestRepository extends JpaRepository<MediaRequest, Integer> {
```

Comment:
Can we use the opportunity to rename the `MediaRequest` entity to `MediaRequestEntity`?